Source/WebKit/NetworkProcess/NetworkSession.cpp

```diff
@@ -182,6 +182,10 @@ NetworkSession::NetworkSession(NetworkProcess& networkProcess, const NetworkSess
 #endif
     , m_dataStoreIdentifier(parameters.dataStoreIdentifier)
 {
+    m_sharedWorkerServer.initLater([](NetworkSession& session, auto& ref) {
+        ref.set(makeUniqueRef<WebSharedWorkerServer>(session));
+    });
```

Comment:
Why not just do 
```
, m_sharedWorkerServer([](NetworkSession& session, auto& ref) {
         ref.set(makeUniqueRef<WebSharedWorkerServer>(session));
     })
```
in the initializers above? Then you don't need the `const` `initLater` above in `LazyUniqueRef`

Reply:
Thanks for the suggestion, you're correct.